control-plane/connect-inject/envoy_sidecar.go

```diff
@@ -96,6 +96,22 @@ func (h *Handler) getContainerSidecarCommand(pod corev1.Pod, multiPortSvcName st
 		cmd = append(cmd, "--base-id", fmt.Sprintf("%d", multiPortSvcIdx))
 	}
 
+	// Check to see if the user has overriden concurrency via an annotation.
+	if pod.Annotations[annotationEnvoyProxyConcurrency] != "" {
+		val, err := strconv.ParseInt(pod.Annotations[annotationEnvoyProxyConcurrency], 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse annotation: %s", annotationEnvoyProxyConcurrency)
+		}
+		if val < 0 {
```

Comment:
I looked in the envoy code and they are not setting an upper limit for the concurrency value (it is a uint32_t) so I think that only checking for >=0 is good enough.